src/features/feedback/ui/FeedbackOverallMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import { HIGH_SCORE, LOW_SCORE, MIDDLE_SCORE } from "@/features/feedback/constants";
+
+interface FeedbackOverallMessageProps {
+    overallScore: number;
+}
+
+const FeedbackOverallMessage = ({ overallScore }: FeedbackOverallMessageProps) => {
+    if (overallScore >= HIGH_SCORE) {
+        return <span>우수한 면접 성과입니다! 자신감을 가지고 면접에 임하세요.</span>;
+    }
+    if (overallScore >= MIDDLE_SCORE) {
+        return (
+            <span>
+                양호한 면접 성과입니다. 조금 더 준비하면 더 좋은 결과를 얻을 수 있을 것입니다.
+            </span>
+        );
+    }
+    if (overallScore >= LOW_SCORE) {
+        return <span>보통 수준의 면접 성과입니다. 부족한 부분을 보완해보세요.</span>;
+    }
+    return <span>더 많은 준비가 필요합니다. 아래 피드백을 참고하여 개선해보세요.</span>;
+};
```

Comment:
```suggestion
const getFeedbackMessage = (score: number): string => {
    if (score >= HIGH_SCORE) {
        return "우수한 면접 성과입니다! 자신감을 가지고 면접에 임하세요.";
    }
    if (score >= MIDDLE_SCORE) {
        return "양호한 면접 성과입니다. 조금 더 준비하면 더 좋은 결과를 얻을 수 있을 것입니다.";
    }
    if (score >= LOW_SCORE) {
        return "보통 수준의 면접 성과입니다. 부족한 부분을 보완해보세요.";
    }
    return "더 많은 준비가 필요합니다. 아래 피드백을 참고하여 개선해보세요.";
};

const FeedbackOverallMessage = ({ overallScore }: FeedbackOverallMessageProps) => {
    return <span>{getFeedbackMessage(overallScore)}</span>;
};
```

이런식으로 함수 분리하면 실제 태그 중복되는 형식을 줄일 수 있을거 같아요